src/Scripting/VisualBasicTest/ObjectFormatterTests.vb

```diff
@@ -76,6 +76,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Scripting.Hosting.UnitTests
             Assert.Equal("Sort { aB=-1, ab=1, Ac=-1, Ad=1, ad=-1, aE=1, aF=-1, AG=1 }", str)
         End Sub
 
+        <Fact>
+        Public Sub Arrays()
+            Dim method = GetType(Sort).GetMethod(NameOf(Sort.ArrayParameter))
+            Dim result = New TestVisualBasicObjectFormatter(maximumLineLength:=100).FormatMethodSignature(method)
```

Comment:
I'd be good to add tests for multi-dimensional arrays as well.

Reply:
Is there an equivalent test for C#?

Reply:
Not sure about C#, so I just added a new test.

Reply:
Just one minor thing: adding the method to `Sort` class is odd. The `Sort` class is used to test ordering of fields. Why not just add a method next to the test method?

Reply:
It should be a public method to avoid the ugly reflection flags. A public method in a test class will be a warning (no test attributes). Maybe add an inner class in the test?

Reply:
I'd prefer using the reflection flags.

Reply:
> Maybe add an inner class in the test?

On second thought - yes, we can add `Signatures` class and put it there. That class can be used for testing other signature shapes in future.

Reply:
I'm guessing the test failures are unrelated.